nautobot/core/tests/test_views.py

```diff
@@ -242,6 +242,25 @@ def test_filtering_on_custom_select_filter_field(self):
         self.assertInHTML(locations[0].name, response_content)
         self.assertInHTML(locations[1].name, response_content)
 
+    def test_filtering_crafted_query_params(self):
```

Comment:
Wonder if we could make this more generic without making it ridiculously complex or just grepping the entire codebase

Reply:
Agreed that would be nice.

We could theoretically add this particular logic to the generic view testing instead of being a single dedicated test, but since the faulty logic was common to all uses of `object_list.html` and involved a single common fix, I didn't feel like that would actually add significant value versus the increased test run time.

I'm always a bit surprised to rediscover that there isn't an `assertNotInHTML` in Django test. It'd be kinda neat to have a set of generic tests that throw various "bad" input into query parameters and assert that that input isn't present in the rendered page, or if present is sanitized appropriately.